qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -811,7 +811,7 @@ def is_controlled(gate):
                     pi2_decomposer = TwoQubitBasisDecomposer(
                         pi2_basis,
                         euler_basis=basis_1q,
-                        basis_fidelity=basis_2q_fidelity,
+                        #                        basis_fidelity=basis_2q_fidelity,
```

Comment:
Oops I forgot about this, I need to come up with a fix for this. In some cases `basis_2q_fidelity` is a dictionary here and that is failing when it gets passed to the rust code because the type is invalid it needs to be `Option<f64>`. I commented this out to workaround it in the short term during development until I could debug what the intent was with the dictionary being passed in is.

Reply:
Fixed by: https://github.com/Qiskit/qiskit/pull/12010/commits/99eaa7b55482942ff5025d81a6f5682fbe72de90